lib/Chunk.js

```diff
@@ -12,18 +12,61 @@ const ERR_CHUNK_ENTRY = "Chunk.entry was removed. Use hasRuntime()";
 const ERR_CHUNK_INITIAL =
 	"Chunk.initial was removed. Use canBeInitial/isOnlyInitial()";
 
+//TODO: Use actual Module type instead of "faked"
```

Comment:
This can be changed now?

Reply:
You're waiting on https://github.com/Microsoft/TypeScript/pull/23570 to be reviewed, iterated, merged, and shipped. I think the associated issue's just marked fixed because we have a candidate fix ready.

Reply:
It's merged now and will be in typescript@next tomorrow.